bin/parse_phenotype.py

```diff
@@ -37,6 +37,24 @@
 assessments = ["BPQ", "TAS", "AQ", "EQ", "STAI",
     "GAD7", "MAIA", "PANAS", "PHQ9", "UCLA", "GSQ"]  # known assessments
 
+BL_one = ["participant_id", "BL_Track_meanacc", "BL_Track_RHO", "BL_Track_meanconf", "BL_Time_meanacc", "BL_Time_RHO",
```

Comment:
These variable names are not interpretable. What does `one` stand for?
Please find out what these variables are.

Reply:
I am not sure what this is. I left it as an ambiguous label so that it could be changed once I knew what it was. Just to be clear this is not a script that I put forward as a final version, but one that I wanted to get onto GITHUB to clear up some of the ambiguity.

Reply:
If you don't know, I am unlikely to have an answer for you. I didn't collect the data. I hope you can find these things out and then complete the script.